src/vcWebFile.cpp

```diff
@@ -7,6 +7,29 @@
 #include "udFileHandler.h"
 #include "udStringUtil.h"
 
+static udResult vcWebFile_Load(udFile *pFile, void **ppBuffer, int64_t *pBufferLength)
+{
+  UDTRACE();
```

Comment:
This is odd; consistent with the rest of this file though

Reply:
Yeah, I thought the same thing but figured I'd just keep it - it's probably because we just copied the udCore HTTP implementation and changed it.